docs/_docs/liquid/tags.md

```diff
@@ -78,7 +78,14 @@ end
 
 In order for the highlighting to show up, you’ll need to include a highlighting
 stylesheet. For Pygments or Rouge you can use a stylesheet for Pygments, you
-can find an example gallery [here](http://help.farbox.com/pygments.html).
+can find an example gallery [here](https://github.com/richleland/pygments-css) or [here](http://jwarby.github.io/jekyll-pygments-themes/languages/ruby.html).
```

Comment:
Let's give a higher priority to the Jekyll-oriented repo instead:

```suggestion
can find an example gallery [here](https://jwarby.github.io/jekyll-pygments-themes/languages/ruby.html) or from [its repository](https://github.com/jwarby/jekyll-pygments-themes).
```

Reply:
That’s fine, the previous example was visual but didn’t have readily available CSS so I kept it first. They are very similar otherwise